contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
On Wed, 2015-11-04 at 14:07 -0800, Scott Dodson wrote:

> We should totally just ship containers via rpms so we can conflict at
> that level.

If we knew how to do that I'd ship 'pause' in an rpm pre-installed on
Atomic Host...